app/src/components/AddProposalPanel.js

```diff
@@ -4,33 +4,35 @@ import styled from 'styled-components'
 
 const AddProposalPanel = ({ onSubmit }) => {
   const [title, setTitle] = useState('')
-  const [description, setDescription] = useState('Lorem ipsum...')
+  const [link, setLink] = useState('')
   const [amount, setAmount] = useState(0)
-  const [beneficiary, setBeneficiary] = useState('')
+  const [recipient, setRecipient] = useState('')
```

Comment:
I know that in the figma the word "Recipient" is used, but we were going to change it by "Beneficiary".